syntax = "proto3";
package core;

service HealthCheckService {
    rpc CheckHealth (HealthCheckRequest) returns (HealthCheckReply);
}

message HealthCheckRequest {
    string subsystem_name = 1;
}

message HealthCheckReply {
    int32 version_major = 1;
    int32 version_minor = 2;
    int32 version_patch = 3;
}
